common/environment/setup/misc.sh

```diff
@@ -13,7 +13,7 @@ GNOME_SITE="https://download.gnome.org/sources"
 KERNEL_SITE="https://www.kernel.org/pub/linux"
 CPAN_SITE="https://www.cpan.org/modules/by-module"
 PYPI_SITE="https://files.pythonhosted.org/packages/source"
-MOZILLA_SITE="https://ftp.mozilla.org/pub"
```

Comment:
ftp.mozilla.org still works for me. why change it? did you check that the new value works for all the other users of MOZILLA_SITE?

Reply:
> ftp.mozilla.org still works for me. why change it? did you check that the new value works for all the other users of MOZILLA_SITE? If this is updated it should also be updated in Manual.md

It was failing on me when I wrote this, seems to be all right again. No need to change if its up again.

Reply:
reverted, done